BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -254,6 +261,22 @@ public class AppProcessor {
             services.errorReporter.log(error: error)
         }
     }
+
+    /// Starts timer to send organization events regularly
+    private func startEventTimer() {
+        sendEventTimer = Timer.scheduledTimer(withTimeInterval: 5, repeats: true) { _ in
```

Comment:
🤔 [In MAUI](https://github.com/bitwarden/mobile/blob/main/src/App/Platforms/iOS/AppDelegate.cs#L467) this is configured to happen every 60seconds for iOS, 5min is for Android. Was this changed to unify the platforms?

Reply:
The ticket I was going off of indicated that it was five minutes, so I was going off of that. The Slack discussions I've been able to dig up don't provide any more illumination for me.

Reply:
Just asked in Slack to see what's the correct expected value.